apps/web/components/InfoCard/DetailedInfoCard.tsx

```
@@ -0,0 +1,217 @@
+      return null
+    }
+
+    return (
+      <Inline space={1}>
+        {tags
+          .map((tag) => {
+            if (!tag) {
+              return null
+            }
+            return (
+              <Tag disabled variant={tag.variant}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)